ui/src/store/modules/sharedDimensions.ts

```diff
@@ -70,6 +80,10 @@ const mutations: MutationTree<SharedDimensionsState> = {
   storeHierarchies (state, collection) {
     state.hierarchies = collection || null
   },
+
+  storeHierarchy (state, hierarchy) {
+    state.hierarchy = hierarchy || null
+  },
```

Comment:
Maybe this now warrants a separate store model for hierarchies?

Reply:
I would say yes